src/types.ts

```diff
@@ -6,8 +6,15 @@ export type Suite = {
   }
 }
 
+export type Registry = {
+  scope?: string;
+  url: string;
+  authToken?: string;
+};
+
 export type NpmConfig = {
   registry?: string;
+  registries?: Registry[];
```

Comment:
So are we intending to deprecate `registry` then? Since `registries` supersedes `registry`.

Reply:
Yes, as `registries` provides the same level of functionality as `registry`.
It also allow to add authentication for non-scoped registries.

Reply:
I annotated the `registry` flag as deprecated, and to be use `registries` instead.